compiler/luci-interpreter/src/kernels/TransposeConv.h

```diff
@@ -52,8 +58,7 @@ class TransposeConv : public KernelWithParams<TransposeConvParams>
   int32_t _padding_width{};
   // The scaling factor from input to output (aka the 'real multiplier') can
   // be represented as a fixed point multiplier plus a left shift.
-  int32_t _output_multiplier = 0;
-  int _output_shift = 0;
+  std::unique_ptr<std::vector<ChannelQuantMultipliers>> _quant_multipliers;
```

Comment:
Is it necessary to use unique_ptr ? Why not just use a vector?

Reply:
You are right, vector is enough. I was careless and left this artifact of some experiments.